v3/manifest_empty_user.yaml

```diff
@@ -0,0 +1,12 @@
+---
+
+# model
+model:
+  version: 3
+
+# object type definitions
+types:
+  user: null
+  admin:
+    relations: null
```

Comment:
What about the case where a relation or permission is null?
```yaml
types:
  thingy:
    relations:
      null_relation: null
    permissions:
      null_permission: null

Reply:
I've added a test for that too